packages/contracts/deploy/helpers.ts

```diff
@@ -1,6 +1,13 @@
 import {promises as fs} from 'fs';
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
 
+export const ENS_ADDRESSES: {[key: string]: string} = {
+  mainnet: '0x314159265dD8dbb310642f98f50C066173C1259b',
+  ropsten: '0x112234455C3a32FD11230C42E7Bccd4A84e02010',
+  rinkeby: '0xe7410170f87102DF0055eB195163A03B7F2Bff4A',
+  goerli: '0x112234455C3a32FD11230C42E7Bccd4A84e02010',
```

Comment:
Probably not worth including  rinkeby and goerli as they will be deprecated after the Ethereum PoS Merge.